src/OBO.NET/OboTerm.fs

```diff
@@ -3,10 +3,12 @@
 open DBXref
 open TermSynonym
 
-open ARCtrl.ISA
-
 open System
 
+open ARCtrl.ISA

```

Comment:
out of curiosity, where is this reference needed?

Reply:
There are IsaOntologyAnnotation functions that use this. I think it's mainly used in ISA.NET and some subsequent projects but that was more like @HLWeil 's business.